### ISSUE TITLE
WPF BlazorWebView TabStop not working

### ISSUE THREAD

**Issue by MichaelPeter** [dotnet/aspnetcore#33382]
### Describe the bug
I played arround with WPF and BlazorWebView, since I know Keyboard control is VERY importaint for powerusers I checked if the tab stop interaction works with the web view:

As you see the tab order should be First the Button, then the Button inside the BlazorWebView and then the Combo Box which drops down over the webside.

But the Website is always last, could there maybe some cross site Tab Control?

PS: Have not yet checked how to make keyboard shortcut interactions.

```
        <StackPanel Grid.Row="0" Orientation="Horizontal">
            <TextBlock VerticalAlignment="Center" Text="ComboBox: " />
            <ComboBox TabIndex="2" Margin="2,5,0,5" Width="80">
                <ComboBoxItem>Item 1</ComboBoxItem>
                <ComboBoxItem>Item 2</ComboBoxItem>
                <ComboBoxItem>Item 3</ComboBoxItem>
                <ComboBoxItem>Item 4</ComboBoxItem>
                <ComboBoxItem>Item 5</ComboBoxItem>
                <ComboBoxItem>Item 6</ComboBoxItem>
            </ComboBox>
            <TextBlock VerticalAlignment="Center" Margin="5,0,0,0" Text="Button Tab Index: " />
            <Button TabIndex="0"  Margin="2,5,0,5">ButtonTabIdx3</Button>
        </StackPanel>
        
        <blazor:BlazorWebView TabIndex="1" IsTabStop="True" Grid.Row="1" HostPage="wwwroot/index.html" Services="{StaticResource services}">
            <blazor:BlazorWebView.RootComponents>
                <blazor:RootComponent Selector="#app" ComponentType="{x:Type local:Counter}" />
            </blazor:BlazorWebView.RootComponents>
        </blazor:BlazorWebView>
```

### To Reproduce
In this sample https://github.com/MichaelPeter/BlazorDesktopWPFTestTabStop/blob/master/README.md
I made it reproducable

### Further technical details
- ASP.NET Core version
- Include the output of `dotnet --info`
- The IDE (VS / VS Code/ VS4Mac) you're running on, and its version
